CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
     * `-ingest-storage.read-consistency`: configures the default read consistency.
     * `-ingest-storage.migration.distributor-send-to-ingesters-enabled`: enabled tee-ing writes to classic ingesters and Kafka, used during a live migration to the new ingest storage architecture.
     * `-ingester.partition-ring.*`: configures partitions ring backend.
+* [ENHANCEMENT] Documentation: Added Azure object store workload identity example configuration
```

Comment:
```suggestion
* [ENHANCEMENT] Documentation: Added Azure object store workload identity example configuration. #8332
```

There's the PR number missing, and can you move this entry to the "Documentation" section below?